cucumber/_authenticators_common/features/support/hooks.rb

```diff
@@ -3,3 +3,30 @@
 Before('@skip') do
   skip_this_scenario
 end
+
+# Reset the DB between each test
+#
+# Prior to this hook, our tests had hidden coupling.  This ensures each test is
+# run independently.
+Before do
```

Comment:
Is this something we have to replicate in the Before hooks for every suite of cuke tests? If so maybe we should file an issue to find a way to DRY it up?

Reply:
Done: https://github.com/cyberark/conjur/issues/1470